interpretation/ww_head.py

```diff
@@ -130,8 +130,10 @@ def process(message, isGameMaster = False):
             members_to_add = check.users(message)
             if is_owner(user_id,channel_id) == False:
                 return [Mailbox().respond("I\'m sorry, but you cannot use this command over here!")]
-            
-            return #TODO
+            command = Mailbox()
+            for x in members_to_add:
+                command.edit_cc(x,channel_id,1)
+            return Mailbox().respond("Insert Randium's comment here")]
```

Comment:
Now again, you're doing nothing with those edit_cc commands that you gave to another instance. It won't edit the channel if you don't return the mailbox that contains those edit_cc commands.
If you replace Mailbox() with command in the last line here, it should work.

Reply:
Oh hahaha sorry about that :)